### ISSUE TITLE
Dead draft?

### ISSUE THREAD

**Issue by brianjmurrell** [autocrypt/protected-headers#30]
I'm guessing by the age of last modifications of this repository that this draft has died?

I'm assuming one of the goals of this proposal was to be able to protect the `Autocrypt:` header from tampering (i.e. MitM attack).  Is that correct?

Is there some subsequent work meant to replace this dead draft in achieving the above goal (among others)?

**Comment by hpk42** [comment]
No, protected headers is/was meant to complement Autocrypt, and is in fact partially implemented in Autocrypt-supporting clients.  The main point about protected headers is defining/specifying which headers get included in an encrypted part of a message.  Most Autocrypt-supporting clients (i think) make sure that the "Subject" header is only in the encrypted part.   Delta Chat says explicitely in its FAQ what it does there https://delta.chat/en/help#message-metadata

At this point, i am not sure if the expired "Protected Headers" draft here (also expired at IETF) makes any sense to consider changing.  So we should probably archive this repo, overall.

**Comment by brianjmurrell** [comment]
But why would one not include the `Autocrypt:` header in the protected header set as it is otherwise vulnerable to compromise in transit, potentially supplying Bob or Alice with an attacker's public key posing as Bob or Alice's?  Doesn't protecting it provide some assurance that it is indeed the legitimate key for Bob or Alice and has not be substituted en route?

Is https://datatracker.ietf.org/doc/draft-ietf-lamps-header-protection/ the current version of what was in this repo?

**Comment by dkg** [comment]
Yes, https://datatracker.ietf.org/doc/draft-ietf-lamps-header-protection/ is the right way forward.  It has much more extensive testing and a principled approach. I'll update this repo to point to the right place.
